### ISSUE TITLE
drugs and reactions

### ISSUE THREAD

**Issue by goodb** [geneontology/pathways2GO#108]
Example R-HSA-186797 

It looks like some reactions e.g. R-HSA-9674095 have drugs in them as inputs/outputs but are not being removed.   (As they should according to https://github.com/geneontology/pathways2GO/issues/84 ) In these cases, the reaction is getting converted but the drugs are just taken out.

**Comment by goodb** [comment]
@ukemi @deustp01 quick check in here before I delve into this.  Are you two still certain that we want to eliminate reactions that involve drugs from the conversion process?  It seems to me that it reduces the value of the resulting GO-CAMs and certainly adds another degree of complexity to the code.  Obvious questions of high value like 'what downstream effects might this drug have' are now impossible to answer with the reactome-generated go-cam knowledge base.  

I think it makes sense to continue keeping the disease pathways out of the conversion process for now because we don't have a good representation for broken genes or processes in GO-CAM.  But could you remind me what about including reactions involving drugs produces a problem for GO-CAM ?

**Comment by ukemi** [comment]
We eliminate them because strictly they are out of scope for GO. The pathways and reactions that include drugs are not 'normal ' pathways and may as a result not reflect the overall 'normal' role of gene products that are also in those pathways or reactions. But I do agree, those and the disease pathways are extremely valuable to our overall mission. BIOMED-CAM. Another grant proposal!

**Comment by ukemi** [comment]
It definitely makes the kinds of questions I've been toying with impossible without asking the query after it has been manually distilled wrt the gene products with which drugs interact.

**Comment by deustp01** [comment]
Naive biologist idea: drugs can be transported and metabolized and do other things as well, but now and for a while to come, the only way that Reactome annotates a drug is to create a reaction in which the drug binds to a protein (normal or genetic mutant) to yield an inert protein - drug complex, and that complex is then annotated as a negative regulator of whatever functions the unbound protein has. As a result of this limitation, no information would be lost if a user was able to view any GO-CAMs involving that protein with the relevant activity units deleted. (This same functionality would enable a user to use GO-CAMs to ask what possible effects of a loss-of-function mutation in a protein are.)

This would avoid discussions of whether there is a space for drugs in GO, but no clue here whether it's practical to implement. It's easy in a Reactome pathway view because we already have display tools to render the normal pathway with big red X's superimposed on the protein(s) whose function has been lost - look at diseases of metabolism for examples.

**Comment by ukemi** [comment]
Great idea, but the scope issue still looms over this I think. I agree with you that one of the major use cases for the models will be to look at downstream effects of perturbations, whether by pharmacological means or genetic means.

**Comment by ukemi** [comment]
Another plus to having this kind of overlay is to expand this discussion to look at regulatory interactions in general, echoing back to the sometimes ATP can be a drug discussion.

**Comment by ukemi** [comment]
I think your arguments are all good. It seems like this should be a PI decision. I can certainly see its utility.

**Comment by judyblake** [comment]
so a question.... how complete are the annotations?  i.e. is the drug data consistently captured in Reactome?  and a general question ... how often would these CAM models be refreshed from Reactome?  what is the expectation?  Is there any sense that we can 'keep up with' drug bindings?  what is REactome's experience with drugs recalled from the market, etc.  oh, maybe that's 'questions'.

**Comment by ukemi** [comment]
The CAM models would be refreshed with every Reactome release. @deustp01 can you shed light on the other questions?

**Comment by goodb** [comment]
Code is ready either way here.  Please advise as it impacts counts needed for manuscript.

**Comment by deustp01** [comment]
@judyblake 

>  is the drug data consistently captured in Reactome?

We are annotating groups of interesting drugs, starting with drugs that affect blood coagulation, now including everything we can find in the literature and that B Shoichet thinks is plausible that affects SARS-CoV-2 infection and host responses to it. So it's a skewed small sample, distinct from operations like CHEMBL or PharmGKG or DrugCentral that aim for comprehensive coverage. So we haven't even addressed the issue of "keeping up".

> what is Reactome's experience with drugs recalled from the market

We strongly prefer to annotate drugs that are approved for clinical use somewhere but this is not an absolute requirement. Unlike those other resources, our target users are researchers - could be translational research - but not medical practitioners. So we in fact do annotate, e.g., some kinase inhibitors that target mutant forms of kinases but that are too toxic for clinical use in humans.

**Comment by ukemi** [comment]
what is Reactome's experience with drugs recalled from the market

Even then, it might be useful to have information about the drugs because they may come into fashion later, thalidomide.

I think @goodb has an excellent point above, if the information is of high quality, is there for the taking and requires effort to ignore, and requires extra effort to filter, then why not have it? We have looked and worked with Reactome enough to know the quality. However, I would not recommend wholesale import of resources without working with them closely first. My 2 cents.

**Comment by Beej4** [comment]
> what is Reactome's experience with drugs recalled from the market

Reactome's focus is approved drugs. Where a group of drugs have pharmacological action on a target, we place that group of drugs into a "defined set" where all drugs are members. There are some occasions where we've curated experimental drugs that are in late stage clinical trials. Since these drugs are not fully approved yet (but the hope is that they will be), we place these drugs in a "candidate set" where approved drugs are members and these experimental drugs are candidates. 

Recalled drugs, if we felt we needed to include them, would fall into a candidate set of drugs. However, we steer clear of such drugs, even as they still display the pharmacological activity at a target, they have been withdrawn for adverse effects.

> 
> Even then, it might be useful to have information about the drugs because they may come into fashion later, thalidomide.

True, but as mentioned above, the focus is on approved drugs.

> 
> I think @goodb has an excellent point above. If the information is of high quality, is there for the taking, requires effort to ignore, then why not have it? We have looked and worked with Reactome enough to know the quality. However, I would not recommend wholesale import of resources without working with them closely first. My 2 cents.

In my humble opinion, the information is of high quality! We started drug curation with cardiovascular drugs (anticoagulants, anti-anginals, antihypertensives, antiarrhythmics). The basic premise is a drug binds a target and the resultant complex is a regulator of the target (positive or negative). This is how simple we'd like to keep drug curation. It doesn't affect the existing Reactome pathway, see it as an overlay of information.

Now we curate drugs for the main targets known to pharma (namely GPCRs and ion channels) because these targets have the biggest reserve of approved drugs and of late, we've also concentrated on potential therapeutics against Covid-19 infection. As we're doing these, we can also be swayed by collaborations which may ask us to curate specific targets so we're very flexible in our drug curation approach.
